packages/manager/apps/web-ongoing-operations/src/pages/update/Update.tsx

```diff
@@ -123,21 +122,18 @@ export default function Update() {
       }
       return { operationID };
     },
-    onSuccess: (data) => {
+    onSuccess: async (data) => {
       const { operationID } = data;
       clearNotifications();
       addSuccess(<OdsText>{t('domain_operations_upload_success')}</OdsText>);
+      await queryClient.invalidateQueries({ queryKey: [taskMeDomain] });
```

Comment:
I think here you might be waiting for both request to refetch in sequence, which is not optimal

you can omit awaiting for it, also you should have a better key management, prefer well formatted array of string instead of array containing single string

you have that : 

```
export const taskMeDomain = '/me/task/domain';
export const taskMeDns = '/me/task/dns';   
```

it's better to declare like this:

```
export const taskMeDomain = ['me', 'task', 'domain'];
export const taskMeDns = ['me', 'task', 'dns']';   
```
with this technique you can do

```
queryClient.invalidateQueries({ queryKey: ['me', 'task'] });   
```
instead of

```
queryClient.invalidateQueries({ queryKey: [taskMeDomain] });
queryClient.invalidateQueries({ queryKey: [taskMeDns] });  
```